packages/aws-cdk-lib/aws-stepfunctions/lib/util.ts

```diff
@@ -1,4 +1,30 @@
+import { AwsOwnedEncryptionConfiguration } from './aokencryptionconfiguration';
+import { CustomerManagedEncryptionConfiguration } from './cmkencryptionconfiguration';
+import { CfnActivity, CfnStateMachine } from './stepfunctions.generated';
+
 export function noEmptyObject<A>(o: Record<string, A>): Record<string, A> | undefined {
   if (Object.keys(o).length === 0) { return undefined; }
   return o;
+}
+
+// eslint-disable-next-line max-len
```

Comment:
Please remove this and format the following line and format the method definition correctly.

Reply:
```ts
export function buildEncryptionConfiguration(encryptionConfiguration? : EncryptionConfiguration): 
  | CfnStateMachine.EncryptionConfigurationProperty
  | CfnActivity.EncryptionConfigurationProperty
  | undefined 
{
...
```

or if that is still too long:

```ts
export function buildEncryptionConfiguration(
  encryptionConfiguration? : EncryptionConfiguration
): 
  | CfnStateMachine.EncryptionConfigurationProperty
  | CfnActivity.EncryptionConfigurationProperty
  | undefined 
{
...
```

Reply:
Thanks will do.

Reply:
Spoke with paulhcsun offline and we will omit the return type since this will be put in `private` and won't be used by customers. We will not run issues with JSII translating unions